taropsbt/interface.go

```diff
@@ -109,6 +114,51 @@ func (i *VInput) Asset() *asset.Asset {
 	return i.asset
 }
 
+// serializeScriptKey serializes the input asset's script key as the PSBT
+// derivation information on the virtual input.
+func (i *VInput) serializeScriptKey(key asset.ScriptKey, coinType uint32) {
+	if key.TweakedScriptKey == nil {
```

Comment:
?

Reply:
Not sure what the question refers to. But for non-local keys, we might not have the information about how the script key was created (meaning we only know the Taproot output key, not the internal key and tweak).